Example/Harmony/Utils/BaseURLRequestAdapter.swift

```diff
@@ -48,6 +52,26 @@ public class BaseURLRequestAdapter: RequestAdapter {
         
         request.url = components.url
         
-        return request
+        completion(.success(request))
+    }
+    
+    public func retry(_ request: Request, for session: Session, dueTo error: Error, completion: @escaping (RetryResult) -> Void) {
+        should(0, session, retry: request, with: error, completion: completion)
+    }
+    
+    private func should(_ index: Int, _ manager: Session, retry request: Request, with error: Error, completion: @escaping (RetryResult) -> Void) {
+        if index < retriers.count {
+            let retrier = retriers[index]
+            retrier.retry(request, for: manager, dueTo: error) { retryResult in
+                switch retryResult {
+                case .retry, .retryWithDelay(_), .doNotRetry:
```

Comment:
Please review this as I had to interpret it from two API versions. I tested the app successfully.

Reply:
For me it's fine